libs/blocks/brick/brick.css

```diff
@@ -338,12 +338,11 @@
 
   .brick.split.row .foreground .brick-media img,
   .brick.split.row .foreground .brick-media video {
-    width: 50%;
+    width: 100%;
     height: 100%;
     object-fit: cover;
     padding: 0;
     margin: 0;
-    position: absolute;
   }
```

Comment:
The brick changes lead to slight layout discrepancies compared to what we have now, I'd suggest we scope these to the preflight styles as well

https://github.com/user-attachments/assets/fbc69911-696c-473f-835d-dc0a1630275b